tox.ini

```diff
@@ -4,7 +4,7 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py27, py37
+envlist = py27, py35, py36
```

Comment:
Including py35 and py36 make sense but why did you remove py37 from the list?

Reply:
To align it with the Travis build. As Travis only tests 27, 35 and 36 I thought it makes sense for you tox to use the same? Could add 37 back?

Reply:
Ah OK, I didn' t have in mind that we only test with 3.6 in Travis. I think that was due to Python3.7 not being available in Travis at that time. We should change that. Means adding py37 test to tox and Travis would be the move here, I think.

Reply:
agreed, ill do that